docs/usability/using_image_validation_tool.md

```diff
@@ -0,0 +1,169 @@
+# PE/COFF Image Validation
+
+The PE/COFF image validation tool is a command line tool used to verify tha
+memory protection requirements such as section alignment and write / execute
+settings are applied correctly. This tool also provides the ability to check,
+set, and clear the NX_COMPAT flag found in OPTIONAL_HEADER.DllCharacteristics.
+
+## Synopsis
+
+image_validation.py [-h] -i FILE [-d] [-s MODULE] [--set-nx-compat SET_NX_COMPAT] [--clear-nx-compat CLEAR_NX_COMPAT] [--get-nx-compat]
```

Comment:
```suggestion
image_validation.py [-h] -i FILE [-d] [-p PROFILE] [--set-nx-compat] [--clear-nx-compat] [--get-nx-compat]
```

Reply:
Thanks, I was a little confused by the previous parameters.